src/Tribe/Repository/Core_Read_Interface.php

```diff
@@ -411,7 +416,7 @@ public function collect();
 	 * @param int  $batch_size       The number of post IDs to fetch at a time when using a generator; ignored
 	 *                               if `$return_generator` is false.
 	 *
-	 * @return array<int>|Generator<int> An array of all the matching post IDs, or a generator of them
+	 * @return array<int>|\Generator<int> An array of all the matching post IDs, or a generator of them
```

Comment:
```suggestion
	 * @return array<int>|Generator<int> An array of all the matching post IDs, or a generator of them
```
import at top with

`use Generator;`